demo/02-customer-infra.sh

```diff
@@ -18,3 +18,29 @@ az network vnet create \
   --subnet-name ${CUSTOMER_VNET_SUBNET1} \
   --subnet-prefixes 10.0.0.0/24 \
   --nsg "${NSG_ID}"  --location ${LOCATION}
+
+az keyvault create \
+  --name "${CUSTOMER_KV_NAME}" \
+  --resource-group "${CUSTOMER_RG_NAME}" \
+  --location "${LOCATION}" \
+  --enable-rbac-authorization true
+
+# Get current user's object ID
+CURRENT_USER_ID=$(az ad signed-in-user show --query id -o tsv)
+
+# Assign Key Vault Crypto Officer role to current user
+az role assignment create \
+  --assignee "${CURRENT_USER_ID}" \
+  --role "Key Vault Crypto Officer" \
```

Comment:
Can we ask the lifecycle team that there are no concerns about using this role?

I understand that there shouldn't as it is scoped to the KV that we are creating, but just in case they have a specific policy about this.

Reply:
Discussed through other channels and it should be fine